SimCore/include/SimCore/Event/EventWeights.h

```diff
@@ -0,0 +1,151 @@
+//
+// Created by Wesley Ketchum on 4/27/24.
+//
+
+#ifndef SIM_CORE_EventWeights_H
+#define SIM_CORE_EventWeights_H
```

Comment:
```suggestion
#ifndef SIMCORE_EventWeights_H
#define SIMCORE_EventWeights_H
```
this is how you did it in other cases too

Reply:
Thanks. Fixed.